### PR TITLE
Migrate RCHTTPStatusCodes

### LINKED ISSUE #579: Migrate RCHTTPStatusCodes

### PR DESCRIPTION
**Pull request by taquitos**
resolved #579

### CODE REVIEW

File: Purchases.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -699,15 +699,16 @@
 			isa = PBXGroup;
 			children = (
 				2CD72940268A820E00BFC976 /* FoundationExtensions */,
-				B3DDB55726854850008CCF23 /* Public */,
```
Comment (taquitos) [inline Purchases.xcodeproj/project.pbxproj:702]: This is a reorganization, alphabetical folders, then files, with the addition of `Networking` folder

**Review by aboedo — COMMENTED** [review]
this will conflict with #509, since that PR also does it. ETags is very close to ready to go, perhaps we should list out the stuff that gets migrated on that PR and assign them to @vegaro, then rebase from it?

**Comment by taquitos** [comment]
@aboedo yeah, I see that conflict, but I think it's ok. #509 will be merged to main so we can ship before Swift migration. I'm happy to resolve this merge conflict after. The conflicts won't be huge.

**Comment by taquitos** [comment]
This has been superseded by #509 and #640

**Pull request closed without merging**